core/src/test/java/org/infinispan/eviction/impl/MarshalledValuesEvictionTest.java

```diff
@@ -67,12 +70,68 @@ public void testEvictPrimitiveKeyCustomValue() {
             expectedReads++; // unmarshall old value if overwritten
          expectedWrites++; // just the value
       }
-      assertEquals(expectedWrites, EvictionPojo.Externalizer.writes.get());
-      assertEquals(expectedReads, EvictionPojo.Externalizer.reads.get());
+      assertEquals(expectedWrites, EvictionPojoMarshaller.writes.get());
+      assertEquals(expectedReads, EvictionPojoMarshaller.reads.get());
+   }
+
+   static class EvictionPojoMarshaller implements org.infinispan.protostream.MessageMarshaller<EvictionPojo> {
```

Comment:
I think you can re-use `CountMarshallingPojo` here and just configure the `org.infinispan.test.TestDataSCI`